### PR TITLE
Allow ballot resubmissions 

### PR DESCRIPTION
**Pull request by nijoe1**
- Allow ballot resubmissions 
- Fixes import ballot bugs
- Opens distribute threshold configuration in distribute page

### CODE REVIEW

File: src/features/ballot/components/BallotAllocationForm/ImportXLSX.tsx [hunk 1]
```diff
@@ -13,6 +13,10 @@ export function ImportXLSX() {
   const save = useSaveBallot();
   const csvInputRef = useRef<HTMLInputElement>(null);
 
+  useEffect(() => {
+    // 
+  }, [votes, setVotes, save]);
+
```
Comment (hussmartinez) [inline src/features/ballot/components/BallotAllocationForm/ImportXLSX.tsx:19]: is this intentional?
what are you using it for?

File: src/features/ballot/components/BallotOverview.tsx [hunk 2]
```diff
@@ -38,9 +43,14 @@ function BallotOverview() {
...
 
   const allocations = ballot?.votes ?? [];
-  const canSubmit = router.route === "/ballot" && allocations.length;
+  const isBallotPage = router.route === "/ballot";
+  const canSubmit = isBallotPage && allocations.length;
   const isProjectsPage = router.route === "/projects";
 
+  useEffect(() => {
+    //
+  }, [ballot]);
```
Comment (hussmartinez) [inline src/features/ballot/components/BallotOverview.tsx:52]: is this intentional?

**Review by hussmartinez — APPROVED** [review]

File: src/features/ballot/components/BallotAllocationForm/ImportXLSX.tsx (hunk 1 above)
Comment (nijoe1) [reply src/features/ballot/components/BallotAllocationForm/ImportXLSX.tsx:19]: While testing this made it more responsive

File: src/features/ballot/components/BallotOverview.tsx (hunk 2 above)
Comment (nijoe1) [reply src/features/ballot/components/BallotOverview.tsx:52]: While testing this made it more responsive

**Pull request merged**